docs/book/src/plugins/available/helm-v1-alpha.md

```diff
@@ -0,0 +1,92 @@
+# Helm Plugin (`helm/v1-alpha`)
```

Comment:
TODO: After I further review I discovered an issue
It is not handle the ConversionWebhook approprieted as the cert manager injection for those CRDs

Reply:
Done/fixed

Reply:
Done